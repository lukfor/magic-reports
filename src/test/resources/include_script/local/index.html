{{include_script("js/index.js")}}

<p class="example">example in red</p>