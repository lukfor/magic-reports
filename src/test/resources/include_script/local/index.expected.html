<!-- js/index.js -->
<script src="include_script_local_files/js/index.js"></script>

<p class="example">example in red</p>